### PR TITLE
[CDU] DIR Page - add distance to waypoint

### PR DESCRIPTION
**Pull request by olivermbs**
**Summary of Changes**
Added distance to selected DIRECT TO waypoint in CDU on Direct To page. 

**Screenshots (if necessary)**

after

**References**
[A320 Displays and Panels site](http://www.a320dp.com/A320_DP/nav-flight-management/sys-14.3.0.html)

**Additional context**

Discord username (if different from GitHub): oivermbs

### CODE REVIEW

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_DirectToPage.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+        let dirETA = "----";
+        let dirDist = "---";
+        let directToLabel = "DIRECT TO[color]blue";
         if (directWaypoint) {
-            directWaypointCell = directWaypoint.ident;
+            directWaypointCell = directWaypoint.ident + "[color]yellow";
         }
         else if (mcdu.flightPlanManager.getDirectToTarget()) {
-            directWaypointCell = mcdu.flightPlanManager.getDirectToTarget().ident;
+            directWaypointCell = "[\xa0\xa0\xa0\xa0\xa0][color]blue";
```
Comment (lousybyte) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_DirectToPage.js:13]: Is ` \xa0` actually working? I know `&nbsp;` doesn't.

Comment (olivermbs) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_DirectToPage.js:13]: Yep, working fine in the sim - makes the spacing between the square brackets closer to the reference material.

Comment (olivermbs) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_DirectToPage.js:13]: Example:
image

Comment (lousybyte) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_DirectToPage.js:13]: Sweet, useful to know for a lot of areas!

**Comment by Ares1378** [comment]
This is the incorrect MCDU page for the Honeywell MCDU. This is a THALES - DIR TO Page. Please use the MCDU Document that was provided in the Discord or message Ares#3394 and I will provide you with the correct reference material and MCDU manual for the honeywell MCDU

**Comment by wpine215** [comment]
Yep, my fault for not mentioning the Honeywell MCDU in the PR template. I added that now, so any future PR's should reference the Honeywell Pegasus Step 1A Rev 0.

**Pull request closed without merging**